orga/app/components/tube/list.js

```diff
@@ -31,6 +31,10 @@ export default class TubeList extends Component {
     return this.args.areas.sortBy('code');
   }
 
+  _transformAttributeToId(tubeId) {
+    return tubeId.replace('tube-', '');
+  }
```

Comment:
```suggestion
  _transformAttributeToId(attribute) {
    return attribute.replace('tube-', '');
  }
```
Peut être on pourrait utiliser un attribut personnaliser pour récupérer directement l'id comme on l'a fait pour les thématiques avec `data-thematics={{tematic.id}}`

Reply:
Oui pourquoi pas, ça clean un peu le code :)